src/Raven.Client/Documents/Operations/ETL/EtlConfiguration.cs

```diff
@@ -107,6 +107,7 @@ public virtual DynamicJsonValue ToJson()
             var result = new DynamicJsonValue
             {
                 [nameof(Name)] = Name,
+                [nameof(TaskId)] = TaskId,
```

Comment:
Why do we need this?

Reply:
I'm fine with that, just wonder why did you need to add it?

Reply:
It fails here after restore:
https://github.com/ravendb/ravendb/blob/462b73c58d4d9d7f48081dc305e053b9cbed3efa/src/Raven.Client/Documents/Operations/ETL/EtlConfiguration.cs#L79

Reply:
Makes sense. After restore `TaskId` should be the same.